src/main/java/org/truffleruby/core/encoding/IsCharacterHeadNode.java

```diff
@@ -21,6 +22,11 @@ public abstract class IsCharacterHeadNode extends RubyBaseNode {
 
     public abstract boolean execute(RubyEncoding enc, AbstractTruffleString string, int byteOffset);
 
+    @NeverDefault
+    public static IsCharacterHeadNode create() {
```

Comment:
Style: `static` methods should be before instance methods, so above the `execute()`

Reply:
Gotcha, fixed.